project/src/main/java/com/google/autograder/servlets/AnswerServlet.java

```diff
@@ -16,7 +16,7 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
 
-@WebServlet("/answer")
+//@WebServlet("/answer")
```

Comment:
Rather than commenting out code, code should be deleted if it's not needed anymore